components/command-arguments.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <ul class="arg-list">
+    <li v-for="(arg, index) in command.args" :key="`arg-${index}`" class="arg">
+      <p class="arg__name">
+        {{ arg.name }} <span>{{ '('+(arg.required?'required':'optional')+')' }}</span>
+      </p>
+      <p class="arg__desc small-text">
+        {{ arg.desc | twoOrphans }}{{ arg.options?':':'.' }}
```

Comment:
`arg.options?':':'.'` is needlessly difficult to read, should be spaces in between operators:
`arg.options ? ':' : '.'`. This seems to be the case in all string concat operations throughout

Reply:
I think this falls within personal preference because I did not have any difficulty reading this with the syntax highlighting from my IDE but I can change this no problem if it makes it easier for others